recipes/effolkronium_random/all/conanfile.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from conans import ConanFile, tools
+
+class RandomConan(ConanFile):
+    name = "effolkronium_random"
```

Comment:
```suggestion
    name = "effolkronium-random"
```

Use `-`, not `_`. There are other projects: bvdberg-ctest, kainjow-mustache, llvm-openmp, mpark-variant, taocpp-json ...

Reply:
`_` is from the cmake lists... On mobile I can grab the link after

Reply:
Yeah, but we already have this discussion for mpark-variant, take a look https://github.com/conan-io/conan-center-index/pull/1269

Reply:
Ahhh okay... The cmake targets need to be configured for that project then.

Would it be possible to have https://github.com/conan-io/conan-center-index/wiki/FAQ#should-reference-names-use---or-_ update to be more clear on the topic? 

> Recipes should stick to the original name of a library as much as possible 

"however sticking with <author>-<project> is the preferred manner"

In this case, everything is written `_` but [vcpkg](https://github.com/microsoft/vcpkg/search?q=effolkronium&unscoped_q=effolkronium) is `-`

Reply:
Indeed it can be. We should move wiki to docs folder soon, so anyone will be able to PR :)